shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/repository/UserRepository.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.avishek.phong.kmmic.domain.repository
+
+import co.nimblehq.avishek.phong.kmmic.domain.model.User
+import kotlinx.coroutines.flow.Flow
+
+interface UserRepository {
+    fun getProfile(): Flow<User>
```

Comment:
How about this name?
```suggestion
    fun getUser(): Flow<User>
```

Reply:
Because it is in `UserRepository` so named it getProfile.